powerauth-nextstep-model/src/main/java/io/getlime/security/powerauth/lib/nextstep/model/entity/ApplicationExtras.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2019 Wultra s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.getlime.security.powerauth.lib.nextstep.model.entity;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Extras for application context for OAuth 2.0 consent screen.
+ *
+ * @author Roman Strobl, roman.strobl@wultra.com
+ */
+public class ApplicationExtras {
```

Comment:
"Application extras" are supposed to be a free map with any information that 3rd party connectors may wish to add. In case we do typed attributes in `ApplicationExtras` class... there is really no sense to wrap them in `ApplicationExtras` class, these could be in `ApplicationContext` object directly. 

I see two ways to proceed:

- 1) Make `extras` attribute of `ApplicationContext` class a `Map<String, Object>`.
- 2) Make `extras` attribute `Map<String, Object>` as in 1) and if we feel that `requestedScopes` and `applicationOwner` are important application context attributes, let's put them to `ApplicationContext`

I am personally for option 1)...

Reply:
I think 1) is fine.